src/javascript/app/pages/trade/digit_ticker.js

```diff
@@ -98,7 +98,7 @@ const DigitTicker = (() => {
 
     // adjust box sizes for mobile
     const adjustBoxSizes = () => {
-        if (el_container.offsetWidth < 360) {
+        if (el_container !== null && el_container.offsetWidth < 360) {
```

Comment:
```suggestion
        if (el_container && el_container.offsetWidth < 360) {
```
it can also cover ,`undefined`,`NaN`, `empty string ("")`, `0`, `false`.